### PR TITLE
websubmit: Avoid error with Send_APP_Mail

### PR DESCRIPTION
**Pull request by chokribr**

### CODE REVIEW

**Comment by tiborsimko** [comment]
See #3600

**Comment by tiborsimko** [comment]
As discussed in the chat room betwenn @chokribr and @ludmilamarian:

a quick fix can be: 

record_owners = print_record(int(sysno), 'tm', \
[CFG_WEBSUBMIT_RECORD_OWNER_EMAIL]).strip()
so int(sysno) instead of sysno
the fix would be in the Send_APP_Mail.py

actually this Send_APP_Mail function doesn't really need to call the print_record (this is some very old code). It can just as simply run get_fieldvalues(sysno, CFG_WEBSUBMIT_RECORD_OWNER_EMAIL) to retrieve the same info

**Comment by tiborsimko** [comment]
@chokribr Would you have an updated PR?

File: modules/websearch/lib/search_engine.py [hunk 1]
```diff
@@ -5017,7 +5017,7 @@ def print_record(recID, format='hb', ot='', ln=CFG_SITE_LANG, decompress=zlib.de
         display_claim_this_paper = False
 
     can_edit_record = False
-    if check_user_can_edit_record(user_info, recID):
+    if  not (format.lower().startswith('t')) and check_user_can_edit_record(user_info, recID):
```
Comment (chokribr) [inline modules/websearch/lib/search_engine.py]: if we are seeking for the content of only one field from metadata, no need to check whether user can or not edit record

**Comment by chokribr** [comment]
ok done #3758, I'll sent another PR to explain  search_engine modifications

**Pull request closed without merging**